generate_brick.py

```diff
@@ -242,6 +242,53 @@ def add_definitions():
             if len(definition[2]):
                 G.add((term, RDFS.seeAlso, URIRef(definition[2])))
 
+    qstr = """
+    select ?param where {
+      ?param rdfs:subClassOf* brick:Parameter.
+    }
+    """
+    params = [row["param"] for row in G.query(qstr)]
+    for param in params:
+        words = param.split("#")[-1].split("_")
+        prefix = words[0]
+        suffix = words[-1]
+        limit_def_template = "A parameter that places {direction} bound on the range of permitted values of a {setpoint}."
```

Comment:
@jbkoh you should be querying for subclasses of `brick:Limit` rather than doing string parsing. In general, all code that operates on the class hierarchy should be working with one of the structured forms (the Python dicts or the RDF model) and not the class names

Reply:
I think you are looking at an outdated commit. I am using `Limit`.